markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 18 Line 1_
```markdown
I get 10 times more traffic from [Google] [1] than from
```
A single comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
a reply to a single comment

Reply:
a replly to a single comment in github